File: output/ChatGPT/codenet/Python/C++/s636719275.cpp
#include <iostream>
#include <string>

using namespace std;

int check(string a, string b) {
    int s = 0;
    for (int i = 0; i < a.length(); i++) {
        s += a[i] != b[i];
    }
    return s;
}

int main() {
    string s, t;
    cin >> s >> t;

    int ans = 1e6;
    for (int i = 0; i < s.length(); i++) {
        string a = s.substr(i);
        if (a.length() >= t.length()) {
            ans = min(ans, check(t, a.substr(0, t.length())));
        }
    }

    cout << ans << endl;

    return 0;
}

File: output/ChatGPT/codenet/Python/Java/s636719275.java
String s = input.nextLine();
String t = input.nextLine();

int check(String a, String b) {
    int s = 0;
    for (int i = 0; i < a.length(); i++) {
        if (a.charAt(i) != b.charAt(i)) {
            s++;
        }
    }
    return s;
}

int ans = 10^6;
for (int i = 0; i < s.length(); i++) {
    String a = s.substring(i);
    if (a.length() >= t.length()) {
        ans = Math.min(ans, check(t, a.substring(0, t.length())));
    }
}
System.out.println(ans);